flow/endpoint.yaml

```diff
@@ -62,17 +62,34 @@ components:
       properties:
         mode:
           description: |-
-            Determines the mapping of tx_names to rx_names.
+            Determines the intent of creating traffic sub-flow(s) between the device 
+            endpoints, from the entities of <b>tx_names</b> to the entities of <b>rx_names</b> 
+            to derive how <b>auto</b> packet fields can be populated with 
+            the actual value(s) by the implementation.
 
-            The mesh mode creates traffic between each value in tx_names to
-            every value in rx_names except in the case where the individual 
-            tx_names value is equal to rx_names value which will be ignored.
-            Identical values in tx_names or rx_names MUST raise an error.
+            The <b>one_to_one</b> mode creates traffic sub-flow(s) between each device endpoint pair in 
+            tx_names to rx_names by index.
+            The length of tx_names and rx_names MUST be the same.
+            Same device name can be repeated multiple times in tx_names or rx_names, in any order to create desired meshing detween device(s).
+            For 2 values in tx_names and 2 values in rx_names, generated device endpoint pairs (each pair representing a traffic sub-flow) would be 2.
```

Comment:
Could rephrase as '...in rx_names , 2 device endpoint pairs would be generated ( each pair representing a traffic sub-flow ) . '

Reply:
Done.